docs/model_extensions.rst

```diff
@@ -37,7 +37,41 @@ Both of the fields are customly defined in Django Extensions as
 ``CreationDateTimeField`` and ``ModificationDateTimeField``.
 Those fields are subclasses of Django's ``DateTimeField`` and will store
 the value of ``django.utils.timezone.now()`` on the model's creation
-and modification, respectively
+and modification, respectively.
+
+The ``TimeStampedModel`` defines following ```Meta`` options:
+1. ``get_latest_by`` on ``modified`` field.
+2. ``ordering`` based on ``-created`` and ``-modified`` fields.
+3. ``indexes`` on ``created``, ``modified`` fields.
+
+Usage:
+
+::
+  class Cheese(TimeStampedModel):
+      name = models.CharField(max_length=50, null=True, blank=True)
+
+To override the ``Meta`` class in the concrete class with new options, then
+``Meta`` of the concrete class should inherit from ``TimeStampedModel.Meta``.
+
+::
+  class Sale(TimeStampedModel):
+      quantity = models.IntegerField()
+
+      class Meta(TimeStampedModel.Meta):
+          db_table = "sale"
+
```

Comment:
When explicitly extending a `Meta` class from abstract model and you do not want abstract class then `abstract = False` should be set explicitly if I'm not mistaken.

Reply:
No Django takes care of that, it sets the flag to `false`, but if you want the child model to be abstract as well then you have to set it to `true` explicitly.

Reply:
Mind blowing. I'm almost sure that had e a case where I had to put `abstract = False` explicitly on a `Meta` which was extending another `Meta`. Probably there was some other reason for that.